### DISCUSSION TITLE
Stat panel color scheme

### DISCUSSION THREAD

**Discussion by rameesponnu** [grafana/grafana#38427] [Platform]
Is it possible to blend multiple colors in stat panel.
![Uploading Screenshot_20210822-015329_WhatsApp.jpg…]()

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
